Fix docs for 0.3 (#399)
Stratio/kind: stratio-docs/en/modules/ROOT/pages/installation.adoc

** xref:attachment$stratio-eks-policy.json[Download permanent permissions for EKS].
** xref:attachment$stratio-aws-temp-policy.json[Download temporary permissions for EKS].

+

> For the EKS deployment you must create the following role manually: "AWSServiceRoleForAmazonEKS" with the associated policy "AmazonEKSServiceRolePolicy".

Reviewer: ```suggestion
For the EKS deployment you must manually create the role "AWSServiceRoleForAmazonEKS" and associate to it the policy "AmazonEKSServiceRolePolicy".
```

> For the EKS deployment you must create the following role manually: "AWSServiceRoleForAmazonEKS" with the associated policy "AmazonEKSServiceRolePolicy" (created by default in AWS).

Reviewer: ```suggestion
For EKS deployment, you must manually create the "AWSServiceRoleForAmazonEKS" role and associate to it the "AmazonEKSServiceRolePolicy" policy (created by default in AWS).
```

+
The currently recommended operating system for this provider is Ubuntu 22.04.

* CloudFormation
+

> In case you have not executed the CloudFormation stack or created the IAM requisites manually previously, you must add the parameter spec:security:aws:create_iam set to true (it is false by default).

Reviewer: ```suggestion
Suppose you have not created the CloudFormation stack or manually created the IAM requirements previously in the account. In that case, you must set the `spec:security:aws:create_iam` parameter to true (default is false).
```

Reviewer 2: Puesto que es un cambio de comportamiento (en 0.2 el valor por defecto es true y ahora pasa a ser false) creo que añadiría un aviso, (o incluso con ese formato de warning, esto como lo vea el equipo de Documentación), más explícito de cambio de comportamiento para que no pase desapercibido.

> In case you have not executed the CloudFormation stack or created the IAM requisites manually previously, you must add the parameter spec:security:aws:create_iam set to true (it is false by default).

Reviewer 2: Puesto que es un cambio de comportamiento (en 0.2 el valor por defecto es true y ahora pasa a ser false) creo que añadiría un aviso, (o incluso con ese formato de warning, esto como lo vea el equipo de Documentación), más explícito de cambio de comportamiento para que no pase desapercibido.
